src/subscriber.ts

```diff
@@ -196,6 +242,20 @@ export class Message {
     }
   }
 
+  /**
+   * Modifies the ack deadline, expecting a response (for exactly once subscriptions).
+   *
+   * @param {number} deadline The number of seconds to extend the deadline.
+   * @private
+   */
+  async modAckWithResponse(deadline: number): Promise<AckResponse> {
+    if (!this._handled) {
```

Comment:
Same point as above: Can we short-circuit this in case exactly once delivery is disabled and return success?